src/store/plugins/modals.js

```diff
@@ -14,7 +14,8 @@ export default (store) => {
     state: { opened: [] },
     mutations: {
       open(state, modal) {
-        state.opened.push(modal);
+        state.opened
+          .push({ ...modal, props: { ...modal.props, body: [].concat(modal.props.body) } });
```

Comment:
Not all modals have a `body`, and some modals can expect a different type of `body`. So, this part shouldn't be aware of props that a specific modal accepts, let's revert this.